tslint.json

```diff
@@ -18,6 +18,7 @@
         "trailing-comma": false,
         "no-string-literal": false,
         "interface-name": false,
+        "max-classes-per-file": [false],
```

Comment:
Can we do this on a per-file basis please? `services/github.ts` shows how. I've done this only for error types, which it doesn't seem useful to create a single class for. If we ever have a class that isn't, and is additional, we'll split them out.